miden/tests/integration/air/chiplets/hasher.rs

```diff
@@ -81,5 +83,5 @@ fn build_mtree_update_test_inputs() -> (Vec<u64>, MerkleSet) {
         tree.depth() as u64,
     ];
 
-    (stack_inputs, tree)
+    (stack_inputs, tree, store, leaves)
```

Comment:
It doesn't seem like we use `tree` or `leaves` anywhere where this function is called. Why do we need to return them?